src/vinylcontrol/vinylcontrolsignalwidget.cpp

```diff
@@ -18,16 +17,15 @@ void VinylControlSignalWidget::setSize(int size) {
     setSizePolicy(QSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed));
     setMinimumSize(size, size);
     setMaximumSize(size, size);
-    m_imageData = new uchar[size * size * 4];
-    m_qImage = QImage(m_imageData, size, size, 0, QImage::Format_ARGB32);
+    m_imageData = std::vector<unsigned char>(size * size * 4);
```

Comment:
`uchar` is a Qt typedef which is also used by their API. While its technically just an `unsigned char`, they could decide to change this in the future which would then break our code. Therefore I'd stick to using `uchar`.